samples/sp_whoIsActive/src/extension.ts

```diff
@@ -13,27 +13,27 @@ import MainController from './controllers/mainController';
 let controllers: ControllerBase[] = [];
 
 export function activate(context: vscode.ExtensionContext): Promise<boolean> {
-    let activations: Promise<boolean>[] = [];
-
-    // Start the main controller
-    let mainController = new MainController(context);
-    controllers.push(mainController);
-    context.subscriptions.push(mainController);
-    activations.push(mainController.activate());
-
-    return Promise.all(activations)
-        .then((results: boolean[]) => {
-            for (let result of results) {
-                if (!result) {
-                    return false;
-                }
-            }
-            return true;
-        });
+	let activations: Promise<boolean>[] = [];
```

Comment:
the space changes are intentional?

Reply:
yes, we are enforcing the indentation and space vs tab rules now, these are causing a lot of compilation errors.